src/ReleaseHistory.md

```diff
@@ -1,6 +1,6 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
-## **v3.2.0** (UNRELEASED)
```

Comment:
>v3.2.0 [](http://example.com/codeflow?start=5&length=6)

I used the current one but I am thinking if we want to use 4.0.0-rc1 instead? I believe we should increase the major number, let me know.

Reply:
I also created another one 
[https://github.com/microsoft/sarif-sdk/pull/2623](https://github.com/microsoft/sarif-sdk/pull/2623)

Reply:
It should definitely be 4.0. Why do we consider it an RC?

Reply:
Why didn't we ship 3.2?

Reply:
was following BinSkim we have 4.0.0 rc1,2,3,4 right now